sdk/subgraph-manager-common/src/interfaces/IArmadaSubgraphManager.ts

```diff
@@ -0,0 +1,13 @@
+import type { IUser, SubgraphProviderType } from '@summerfi/sdk-common'
+import { IManagerWithProviders } from '@summerfi/sdk-server-common'
+import type { PositionsByAddressQuery } from '../generated/client'
+import type { IArmadaSubgraphProvider } from './IArmadaSubgraphProvider'
+
+/**
+ * @name IArmadaSubgraphManager
+ * @description interface for the Armada subgraph manager which will be distinct from the DPM subgraph manager
+ */
+export interface IArmadaSubgraphManager
```

Comment:
Good question! On the beginning I was thinking to have single subgraph manager for both armada and borrow, but soon it became obvious they should be separate because of different data and I left it on that way, but now I would lean towards simplifying that and removing providers. Let me know if you agree, thanks 🙏

Reply:
Done :)

Reply:
Thanks!